### DISCUSSION TITLE
Allow editing the site name below the big blue button

### DISCUSSION THREAD

**Discussion by jidanni** [uBlockOrigin/uBlock-issues#1837] [Ideas] [answered]
![20211123T225710](https://user-images.githubusercontent.com/1230959/143077084-53c0a97a-95f4-41cf-8427-49ca92afd176.jpg)
See the line community.windy.com ?
Well, it should be editable.
That would allow the user to shorten it,
to just windy.com, allowing him to whitelist the entire domain very very
easily.
Yes, there are other ways to do the job.

Or, add a button: "[X] entire domain (windy.com)".

**Comment by uBlock-user** [comment, marked as answer]
https://github.com/gorhill/uBlock/issues/3297

**Reply by jidanni** [reply]
OK.
As that issue seems stuck, a workaround would be:
User should now browse just the domain name,
and then hit the big blue button there.
